JitsiConference.js

```diff
@@ -463,6 +464,8 @@ JitsiConference.prototype.leave = function() {
             XMPPEvents.CONFERENCE_PROPERTIES_CHANGED,
             this._updateProperties);
 
+        this.eventManager.removeXMPPListeners();
```

Comment:
`setupXMPPListeners` adds many more listeners than this function removes. I'd say it should undo everything `setupXMPPListeners` did.